packages/gestalt-codemods/4.0.0-4.1.0/touchable-to-taparea.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Converts
+ *  <Touchable onTouch={...} />
+ * to
+ *  <TapArea onTap={...} />
+ */
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+  const src = j(file.source);
+  let touchableLocalIdentifierName;
+
+  src.find(j.ImportDeclaration).forEach(path => {
+    const decl = path.node;
+    if (decl.source.value !== 'gestalt') {
+      return;
+    }
+    const touchableSpecifier = decl.specifiers.find(
+      node => node.imported.name === 'Touchable'
+    );
+
+    if (!touchableSpecifier) {
+      return;
+    }
+
+    touchableLocalIdentifierName =
+      touchableSpecifier && touchableSpecifier.local.name;
+
+    const newSpecifiers = [
+      // Strip out Tooltip import
```

Comment:
@jackhsu978 should we update this comment?

Reply:
Thanks for catching! Done!